docker-build/src/test/java/io/quarkus/ts/docker/DockerBuildIT.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.ts.docker;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.dockerjava.api.model.Image;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.utils.DockerUtils;
+
+@QuarkusScenario
+@DisabledOnNative // in order to avoid extra test time. In my opinion native mode doesn't add value for this scenario
```

Comment:
If the test is disabled on Native, do we still need https://github.com/quarkus-qe/quarkus-test-suite/pull/438/files#diff-873490bdd3f9d04392641dd37b052242fb566f181517f6f5fabeabdce22e85a7 ?

Reply:
these files are generated by `mvn io.quarkus.platform:quarkus-maven-plugin:2.6.1.Final:create` command by default. It's true that we are not going to use it, but I prefer to leave it as it is.